Synapse3.SynapseModule/Patches/PlayerPatches/DeathPatches.cs

```diff
@@ -32,15 +32,21 @@ public static bool PreDeath(PlayerStats __instance, DamageHandlerBase handler)
                 attacker = Synapse.Get<PlayerService>().Players
                     .FirstOrDefault(x => x.ScpController.Scp106.PlayersInPocket.Contains(player));
 
-            var ev = new DeathEvent(player, true, attacker, damageType, damage);
+            var ev = new DeathEvent(player, true, attacker, damageType, damage, null);
             Synapse.Get<PlayerEvents>().Death.Raise(ev);
 
+            if (ev.DeathMesasge != null)
+                handler = new CustomReasonDamageHandler(ev.DeathMesasge);
+
```

Comment:
You can remove this since there is a duplicate down below

Reply:
ok